tests/conftest.py

```diff
@@ -2910,6 +2915,22 @@ def ns_resources_cleanup():
     return _create_ns_resources
 
 
+@pytest.fixture()
```

Comment:
Please add a `session`-scoped factory
You can base it on `bucket_factory()` and `bucket_factory_session()` (make sure to also use the session-scoped fixtures in the parameters)

Reply:
added according to backingstore_factory